pkg/sql/colexec/colexecmisc/COLEXECMISC.bzl

```diff
@@ -0,0 +1,58 @@
+# NB: The helpers here were grafted from pkg/sql/colexec/COLEXEC.bzl.
```

Comment:
I think this is too much code duplication. The helpers should be refactored into a common place. Let me know if you need help with this.